symphony/app/fbcnms-projects/workflows/src/proxy/transformers/metadata-taskdef.js

```diff
@@ -141,19 +141,17 @@ const getTaskdefByNameBefore: BeforeFun = (
 };
 
 const getTaskdefByNameAfter: AfterFun = (tenantId, req, respObj, res) => {
-  if (res.statusCode == 200) {
```

Comment:
Why don't we want to check for a 200?

Reply:
I changed the logic so that only successful responses from conductor trigger the AfterFun. See proxy.js:58 (`proxyRes.statusCode >= 200`)